tests/integration/plusminus/README.rst

```diff
@@ -0,0 +1,37 @@
+=========
```

Comment:
To be removed

Reply:
I kept it to explain what this example is doing.